src/env.cc

```diff
@@ -617,7 +617,7 @@ Local<Value> Environment::GetNow() {
   CHECK_GE(now, timer_base());
   now -= timer_base();
   if (now <= 0xffffffff)
```

Comment:
Does this mean that the bug still exists for numbers in `2^31 .. 2^32` range?

Reply:
@nkbt This PR fixed the bug for those numbers – all others should remain the same

Reply:
Thanks, I was not sure if NewFromUnsigned does exactly that